packages/OptionButton/src/option-button.scss

```diff
@@ -0,0 +1,145 @@
+
+  /* Text */
+  --ids-option-button-text-container-padding: #{tokens.$space-1} #{tokens.$space-3};
+  --ids-option-button-text-color: #{tokens.$grey-800};
+  --ids-option-button-text-font-size: #{tokens.$font-size-5};
+  --ids-option-button-text-font-weight: #{tokens.$font-weight-semi-bold};
+  --ids-option-button-text-line-height: #{tokens.$line-height-xl};
+
+  /* Description */
+  --ids-option-button-desc-color: #{tokens.$grey-600};
+  --ids-option-button-desc-font-size: #{tokens.$font-size-3};
+  --ids-option-button-desc-line-height: #{tokens.$line-height-xxl};
+}
+
+%active-icon-container {
+  background: var(--ids-option-button-icon-container-background);
+}
+
+%disabled-icon-container {
+  background: var(--ids-option-button-icon-container-disabled-background);
+}
+
+.ids-option-button {
+  display: block;
+  position: relative;
+
+  &__radio {
+    opacity: 0;
+    position: absolute;
+  }
+
+  &__label {
+    align-items: center;
+    border: 1px solid var(--ids-option-button-border-color);
+    border-radius: var(--ids-option-button-border-radius);
+    cursor: pointer;
+    display: flex;
+    font-family: var(--ids-option-button-font-family);
+    outline: none;
+    overflow: hidden;
+  }
+
+  &__radio:hover ~ &__label {
+    border-color: var(--ids-option-button-hover-border-color);
+  }
+
+  &__radio:focus ~ &__label {
+    outline: 1.5px solid var(--ids-option-button-focus-outline-color);
+    outline-offset: 2px;
+  }
+
+  &__radio:checked ~ &__label {
+    border: 1px solid var(--ids-option-button-border-color);
+    box-shadow: var(--ids-option-button-checked-box-shadow);
+    cursor: default;
+    outline: none;
+  }
+
+  &__radio:disabled ~ &__label {
+    border: 1px solid var(--ids-option-button-border-color);
+    box-shadow: none;
+    cursor: not-allowed;
+    outline: none;
+  }
+
+  &--unchecked &__radio:checked ~ &__label {
+    box-shadow: none;
+  }
+
+  &__icon-container {
+    @extend %active-icon-container;
+
+    align-items: center;
+    align-self: stretch;
+    display: flex;
+    flex: 0 0 auto;
+    justify-content: center;
+    min-height: var(--ids-option-button-icon-container-height);
+    position: relative;
+    width: var(--ids-option-button-icon-container-width);
+  }
+
+  &--unchecked &__icon-container {
+    @extend %disabled-icon-container;
+  }
+
+  &__radio:checked ~ &__label &__icon-container {
+    @extend %active-icon-container;
+  }
+
+  &__radio:disabled ~ &__label &__icon-container,
+  &--unchecked &__radio:checked ~ &__label &__icon-container {
+    @extend %disabled-icon-container;
+  }
+
+  &__text-container {
+    flex: 1 1 auto;
+    min-width: 0;
+    padding: var(--ids-option-button-text-container-padding);
+  }
+
+  &__text {
+    color: var(--ids-option-button-text-color);
+    display: block;
+    font-size: var(--ids-option-button-text-font-size);
+    font-weight: var(--ids-option-button-text-font-weight);
+    overflow: hidden;
```

Comment:
you can group the declarations for the ellipsis in a class `%ellipsis` and repeat it in `&__text` and `&__desc`